packages/discovery-provider/ddl/migrations/0096_update_comment_threads.sql

```diff
@@ -0,0 +1,18 @@
+begin;
+
+alter table comment_threads
+add column if not exists created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+alter table comment_threads
+add column if not exists updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+alter table comment_threads
+add column if not exists txhash TEXT NOT NULL;
+
+alter table comment_threads
+add column if not exists blockhash TEXT NOT NULL;
+
+alter table comment_threads
+add column if not exists blocknumber integer REFERENCES blocks (number) ON DELETE CASCADE;
```

Comment:
is there a need for these columns? i was basing comment_threads off the remixes table. things can get hairy w this cascading delete and block reverts

Reply:
the reason i added it was for notifications seemingly wanting blocknumber, created_at and a bunch of other stuff, what do you think we should do?